source/tyr/tyr/tasks.py

```diff
@@ -252,6 +254,10 @@ def files_type(files):
             return 'cosmogony'
         if len(files) == 1 and files[0].endswith('.pbf'):
             return 'osm'
+        # OpenAddresses files does not have a predefined naming,
+        # so we check it last, and consider all csv as OA
+        if any(f for f in files if OPEN_ADDRESSES_REGEXP.match(f)):
```

Comment:
There is no possibility of confusing with a NTFS file ?

Reply:
nop they are not dropped in the same directory (the method is `type_of_autocomplete_data`)